docs/modelserving/v1beta1/lightgbm/README.md

```diff
@@ -414,3 +353,38 @@ grpcurl \
       ]
     }
     ```
+## Test the model locally with mlserver
+Once you've got your model serialized `model.bst`, we can then use [MLServer](https://github.com/SeldonIO/MLServer) which implements the KServe V2 inference protocol to spin up a local server. For more details on MLServer, please check the [LightGBM example doc](https://github.com/SeldonIO/MLServer/blob/master/docs/examples/lightgbm/README.md).
```

Comment:
@sivanantha321 let’s also update the local development doc with KServe sklearn runtime to be consistent ?

Reply:
Done. Also added the runtime priority docs